### PR TITLE
Add config for jaeger operator for hypershift

### PR DESCRIPTION
**Pull request by smpsgill**
Adding this config for Jaeger operator install in a ROSA hypershift cluster.

Test repo: https://github.com/jaegertracing/jaeger-operator

### CODE REVIEW

**Review by calebevans — APPROVED** [review]
/lgtm

File: ci-operator/config/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main__jaeger-product-hypershift-4.13-lp-rosa-hypershift.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: jaeger-product-rosa-hypershift
+  cron: 0 10 * * 1
+  remote_api: true
```
Comment (ascerra) [inline ci-operator/config/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main__jaeger-product-hypershift-4.13-lp-rosa-hypershift.yaml:27]: This line can be removed

File: ci-operator/config/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main__jaeger-product-hypershift-4.13-lp-rosa-hypershift.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI"},
+          {"step": "cucushift-hypershift-*", "failure_type": "all", "classification": "Cluster Health Check", "jira_project": "LPTOCPCI"}
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      OCM_LOGIN_ENV: staging
+      OPENSHIFT_VERSION: "4.13"
+      OPERATORS: |
+        [
+            {"name": "jaeger-product", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators"}
```
Comment (ascerra) [inline ci-operator/config/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main__jaeger-product-hypershift-4.13-lp-rosa-hypershift.yaml:46]: I can't confirm or deny at the moment that what you have here will work we'll see what happens in the rehearsal job but I have tested this config for the operator install and it did work

```{"name": "jaeger-product", "source": "redhat-operators", "channel": "!default", "install_namespace": "openshift-distributed-tracing", "operator_group": "jaeger-operator-group"}```

Comment (Simranpal) [reply ci-operator/config/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main__jaeger-product-hypershift-4.13-lp-rosa-hypershift.yaml:46]: @ascerra I think this should work. But I was bit confused about why PQE is installing the operator this way
```suggestion
            {"name": "jaeger-product", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators"}https://github.com/openshift/release/pull/40442/commits/2c27c5d036950e2611c29be9d909914531b07de0#diff-112ea49c1cd2a5af172ba25b6de85ab5679955bc55814e9cd3f3113428cb47a6
```
for onboarding

File: ci-operator/config/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main__jaeger-product-hypershift-4.13-lp-rosa-hypershift.yaml (hunk 1 above)
Comment (Simranpal) [reply ci-operator/config/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main__jaeger-product-hypershift-4.13-lp-rosa-hypershift.yaml:27]: I have removed this line

**Pull request merged**